Userland/allocate.cpp

```diff
@@ -33,18 +33,19 @@
 
 static void usage(void)
 {
-    printf("usage: allocate [number [unit (B/KB/MB)]]\n");
+    printf("usage: allocate [number [unit (B/kiB/MiB)]]\n");
     exit(1);
 }
 
 enum class Unit { Bytes,
```

Comment:
I dunno why the reformatting failed here, but...
```suggestion
enum class Unit {
Bytes,
```

Reply:
Done.